src/main/java/com/unstoppabledomains/resolution/naming/service/ENS.java

```diff
@@ -92,6 +96,30 @@ public String getOwner(String domain) throws NamingServiceException {
     throw new NamingServiceException(NSExceptionCode.NotImplemented, new NSExceptionParams("m|n", "getTokensOwnedBy", getType().toString()));
   }
 
+  @Override
+  public Map<String, Location> getLocations(String... domains) throws NamingServiceException {
+    Map<String, Location> locations = new HashMap<>();
```

Comment:
I think we could throw an exception for ENS and ZNS since they considered as deprecated. Eventually we will remove ENS support and will migrate .zil domains into UNS.

Reply:
I added an unsupported exception for ZNS and ENS